plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -371,6 +389,7 @@ func init() {
 			ServiceAddress: ":8080",
 			TimeFunc:       time.Now,
 			Path:           "/telegraf",
+			Paths:          []string{"/telegraf"},
```

Comment:
You need to remove `Path` here as otherwise `Paths` has a duplicate entry.

Reply:
You mean entire line (it requires to change tests) or just to set it to empty string?

Reply:
nevermind, fixed :)